site/cds_rdm/inspire_harvester/writer.py

```diff
@@ -30,81 +32,126 @@ def _write_entry(self, entry, *args, **kwargs):
 
         existing_records_hits = existing_records.to_dict()["hits"]["hits"]
         existing_records_ids = [hit["id"] for hit in existing_records_hits]
+        current_app.logger.debug(
```

Comment:
```suggestion
        current_app.logger.info(
```
this is more valuable than only debug, we will have a nice list of updates reported